src/java.base/share/classes/jdk/internal/foreign/abi/s390/zos/ZosS390CallArranger.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (c) 2022, 2024, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2023 IBM Corp. All rights reserved.
```

Comment:
This line seems inappropriate given the separate IBM copyright notice below (also in `ZosS390Linker.java`).

I see the same line in other `s390/**/*.java` files, but I don't think it should be there either (we can deal with those files separately).

Reply:
That's because our previous code on zLinux was totally replaced by the code offered in OpenJDK, in which case the copyright was directly copied from there. So we probably have to leave the copyright of code on zLinux unchanged if there is no our modification.

Reply:
Removed the copyright (mentioned above) in the newly added files on z/OS.